news/2018-03-13-project-loved-week-of-march-11th.md

```diff
@@ -0,0 +1,244 @@
+
+[![](/wiki/shared/news/2018-03-13-project-loved-week-of-march-11th/osu/fks.jpg)](https://osu.ppy.sh/community/forums/topics/715022)
+
+From the famed top player [Azer](https://osu.ppy.sh/users/2155578) comes [F.K.S.](https://osu.ppy.sh/beatmapsets/276313), a well known technical map of a Sound Voltex song! As you'd probably expect from both the song and the mapper, the map features challenging rhythms and tricky patterns while still remaining accessible to most players due to its lower star rating. Be sure to check out [captin](https://osu.ppy.sh/users/689997)'s absolutely masterful difficulty featuring tricky cutstreams as well, it's one of my favourite captin maps and kind of a hidden gem seeing as it's a guest difficulty!
+
+[![](/wiki/shared/news/2018-03-13-project-loved-week-of-march-11th/osu/bylas-dla-mnie-wszystkim.jpg)](https://osu.ppy.sh/community/forums/topics/715023)
+
+From notable mapper [fartownik](https://osu.ppy.sh/users/56917) comes the highly coveted Polish meme map [Bylas dla mnie wszystkim](https://osu.ppy.sh/beatmapsets/555030), which has been highly requested through the [Loved Submission form](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform) (which you should all be using to submit maps by the way)! The map features high velocity sliders and tricky cross screen jumps, as well a different **completely related** background for each guest difficulty, all of which are by various Polish mappers and players.  Be sure to check out each difficulty, as they all are unique in their own way and offer different levels of challenge for a wide variety of players.
+
+[![](/wiki/shared/news/2018-03-13-project-loved-week-of-march-11th/osu/walk-this-way.jpg)](https://osu.ppy.sh/community/forums/topics/715024)
+
+From the extremely popular high star mapper [Settia](https://osu.ppy.sh/users/2445897) comes [Walk This Way](https://osu.ppy.sh/beatmapsets/339394), an appropriately challenging beatmap that will push your aim to its limits! The map features highly structured geometric jumps which are both comfortable to play as well as satisfying to hit. Are you strong enough to clear this brutal map 260 BPM map?
+
+[![](/wiki/shared/news/2018-03-13-project-loved-week-of-march-11th/osu/aura.jpg)](https://osu.ppy.sh/community/forums/topics/715025)
+
+People who don't know who [Midge](https://osu.ppy.sh/users/2226436) is or have never seen his maps may not fully grasp just how momentus of an occasion this map being up for the Loved vote is. [aura](https://osu.ppy.sh/beatmapsets/281721) is a unique technical map which was shunted and denied ranking at the time of its creation due to its unusual aesthetics and its stylistically extreme flow and appearance. However, this map has endured as a classic set still today, and more than deserves its leaderboard all this time later: #loveaura.
```

Comment:
google is telling me that `momentus` should be `momentous` ... unless we're following urban dictionary definition